tests/test_visitors/test_filenames/test_wrong_module_name/test_module_magic_name.py

```diff
@@ -31,3 +32,16 @@ def test_simple_filename(assert_errors, filename, default_options):
     visitor.run()
 
     assert_errors(visitor, [WrongModuleMagicNameViolation])
+
+
+@pytest.mark.parametrize('filename', [
```

Comment:
Please, add happy path tests. I mean that we should test that `version2.py` and `version2_8.py` are correct names.

Reply:
Sure